runtime/ms_rest_azure/lib/ms_rest_azure/azure_service_client.rb

```diff
@@ -19,6 +19,8 @@ def initialize(credentials, options = nil)
       @request_headers =  {
           'Content-Type' => 'application/json;charset=utf-8' # This is the current default for Azure services, and content-type supported by Autorest
       }
+      add_user_agent_information("ms_rest_azure/#{MsRestAzure::VERSION}")
+      add_user_agent_information("Azure-SDK-For-Ruby")
```

Comment:
Sounds good, the string looks redundant but it's needed for data collection/analysis and consistency, so I'm fine having it in the runtime. Thanks!